zb-db/src/main/java/io/camunda/zeebe/db/impl/rocksdb/transaction/ZeebeTransactionDb.java

```diff
@@ -38,6 +41,19 @@ public class ZeebeTransactionDb<ColumnFamilyNames extends Enum<ColumnFamilyNames
   private static final Logger LOG = Loggers.DB_LOGGER;
   private static final String ERROR_MESSAGE_CLOSE_RESOURCE =
       "Expected to close RocksDB resource successfully, but exception was thrown. Will continue to close remaining resources.";
+
+  private static Method makeDefaultColumnFamilyHandle;
+
+  static {
+    RocksDB.loadLibrary();
+
+    try {
+      resolveMakeDefaultColumnFamilyHandle();
```

Comment:
:wrench: Can we keep the reflection of RocksDB internals into `RocksDbInternal.java`?